pkg/model/serializer.go

```diff
@@ -768,7 +767,7 @@ type manifestRaw struct {
 	Icon          string                                   `json:"icon,omitempty" yaml:"icon,omitempty"`
 	Deploy        *DeployInfo                              `json:"deploy,omitempty" yaml:"deploy,omitempty"`
 	Dev           ManifestDevs                             `json:"dev,omitempty" yaml:"dev,omitempty"`
-	Destroy       []DeployCommand                          `json:"destroy,omitempty" yaml:"destroy,omitempty"`
+	Destroy       *DestroyInfo                             `json:"destroy,omitempty" yaml:"destroy,omitempty"`
```

Comment:
Do we need it to be a pointer? I think we are only getting information and not modifying it.

Also, Can we add a unit test for this?

Reply:
We are using it as a pointer following `DeployIngo` approach. Anyway, we are calling `manifest.ExpandEnvVars()` wich modifies `deploy.image`